cfpb/consumerfinance.gov
# Fix error caused by missing PFC image; fix legacy breadcrumb styling

The moving of old PFC pages to this repo in #6307 did not bring with it one image, `Page_icon.png` that is referenced in a commented HTML `img` tag. Django still looks for that image and outputs its path inside the HTML comment, causing a 500 error if someone tries to load that page in a production environment.

The fix proposed is to use a Jinja comment around the `static` tag, leaving the reference there for consideration in future work on this template.

This PR also fixes the breadcrumb styling on these older pages migrated in #6307, and updates the Gulp config so that gulp-newer correctly triggers a rebuild when `.css` files are changed.


---


## Changes

- Puts `static` tag referencing missing image in Jinja comments so that it doesn't cause a 500 error in production environments
- Fix legacy PFC pages' hero breadcrumb styling
- Tell gulp-newer to look for unprocessed `.css` files (in addition to `.less` files) when determining if the styles should be rebuilt


## How to test this PR

1. Run local server with `DEBUG=False`
1. Open http://localhost:8000/paying-for-college/repay-student-debt/ and see a 500 error
1. Pull branch
1. Reload http://localhost:8000/paying-for-college/repay-student-debt/ and see that the page loads, but has janky breadcrumb styling
1. Run `gulp styles:apps`
1. Reload http://localhost:8000/paying-for-college/repay-student-debt/ and see the correct breadcrumb styling


## Screenshots

### Before

![Before: bad breadcrumb styling](https://user-images.githubusercontent.com/1044670/111350792-e14b8900-8658-11eb-902f-46753183682d.png)

### After

![After: good breadcrumb styling](https://user-images.githubusercontent.com/1044670/111350842-ec9eb480-8658-11eb-995c-aecef5daddf3.png)


## Notes

- I am not sure why `tox -e validate-assets` does not return an error (prior to the change in this PR) for the missing image.


## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".

### Front-end testing



#### Browser testing

Visually tested in the following supported browsers:
